### ISSUE TITLE
Feature Request: Support `--config=parallel` for `generate:suite`

### ISSUE THREAD

**Issue by lbajsarowicz** [magento/magento2-functional-testing-framework#653]
### Acceptance Critirea
1. Command `generate:suite` should support `--config=parallel`
2. When suite is generated it is split to couple of groups able to be executed in parallel

### Note:
This can be done as of right now. `generate:suite` command should mimic that behavior:
`bin/mftf generate:tests --config=parallel --tests="{suite:["SomeSuite"]}"`

**Comment by jilu1** [comment]
Internal ticket https://jira.corp.magento.com/browse/MQE-2238 is created

**Comment by okolesnyk** [comment]
Hi @lbajsarowicz 
We will close this issue as there is an existing workaround using generate:tests command.
Please feel free to submit new issue or create PR in case proposed solution isn't a solution in this case.
